README.md

```diff
@@ -72,6 +72,12 @@ Where:
 * `sdkId` is the sdk identifier generated by Yoti Dashboard when you create (and then publish) your app. _Note this is not your Application Identifier which is needed by your clientside code_
 * `path/to/your-application-pem-file.pem` - When you create your app on Yoti Dashboard, your browser generates a .pem file. This is the path to that file.
 
+You'll need to add the `Callback URL` of your site to the integration settings section on your [Yoti Applications page](https://www.yoti.com/dashboard/applications). This is where the user will be redirected to upon successful authentication. For the Example project, you should append `/account/connect` to the end of this URL.
+
+For debugging purposes, you can use [ngrok](https://ngrok.com/) to create a secure tunnel to localhost by running the following command, and adding the https address generated to your callback URl:
```

Comment:
I'd added ngrok to the Ruby SDK Readme before the Dashboard allowed localhost URLs. I will remove this from the Readme and I don't think it should be mentioned here as well. This will also become less relevant once we have a sandbox app.

Reply:
Removed